flipperdevices/flipperzero-firmware
# [FL-1720] BLE GUI refactoring

# What's new

- Separate BLE on service, settings application and debug application (for certification)
- Store BLE settings (now only Enable / Disable) on internal storage
- Rework debug application GUI
- Accessor: move PA6 to PA4
- FuriHal: add assert to gpio callback set

# Verification 

- Enable / disable BLE in Settings. Verify that settings is successfully saved on and loaded from internal storage
- Verify correct debug application behavior (with 2 flippers and SDR)
- Accessor is not causing down button to stop working anymore

# Checklist (do not modify)
